cli/cmd/staking.go

```diff
@@ -311,3 +424,41 @@ func unjailValidator(ctx context.Context, cfg unjailConfig) error {
 
 	return nil
 }
+
+// setupClients is a test helper that creates the omni evm client,
+// omni consensus client and a backend set with the operator private key.
+func setupClients(
+	evmRPC string,
+	network netconf.ID,
+	operatorPriv *ecdsa.PrivateKey,
+) (ethclient.Client, cchain.Provider, *ethbackend.Backend, error) {
+	chainID := network.Static().OmniExecutionChainID
+	chainMeta, ok := evmchain.MetadataByID(chainID)
+	if !ok {
+		return nil, nil, nil, errors.New("chain metadata not found")
+	}
+
+	if evmRPC == "" {
+		evmRPC = network.Static().ExecutionRPC()
+	}
+
+	cprov, err := provider.Dial(network)
```

Comment:
That is actually more problematic than it seems. The problem is it's being used in a lot of places and if we panic it fails a lot of tests. The other thing is that after investigating the consensus rpc default is correct for devnet, and if a test wants to specify it differently it can by creating the client manually. I did that first but then reverted this changes because I see all the issue it causes in other tests as well as figured out the default is the same as I was setting. So it didn't make sense to change this.

Reply:
I still think we should allow setting a `--consensus-rpc` since users might want to use their own halo nodes.